src/bundle/Resources/translations/messages.en.xliff

```diff
@@ -608,6 +608,11 @@
         <target state="new">Delete</target>
         <note>key: section.delete</note>
       </trans-unit>
+      <trans-unit id="1f9b6e37416ff607de9d4fa280dce2f734ea622c" resname="section.list">
+        <source>Sections</source>
+        <target state="new">Sections</target>
+        <note>key: section.list</note>
```

Comment:
My guess is, that this should be under `section` translation domain.
But probably this is outside the scope of this task...